packages/fxa-settings/src/components/Settings/HeaderLockup/index.test.tsx

```diff
@@ -43,7 +45,9 @@ describe('HeaderLockup', () => {
 
   it('emits Glean event on help link click', async () => {
     renderWithLocalizationProvider(<HeaderLockup />);
-    await userEvent.click(screen.getByRole('link', { name: 'Help' }));
+    await userEvent.click(
+      screen.getByRole('link', { name: 'Help Opens in new window' })
+    );
```

Comment:
Doesn't seem to be dep upgrade related?

Reply:
As above, certain upgraded packages seemed to make tests "more aware" – in this case it was trying to match 'Help', but there was an extra `<span>` for screenreaders that was added ("Opens in new window") and now included in the match.